src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -17,20 +17,8 @@
  */
 package org.fcrepo.importexport;
 
-import org.apache.commons.cli.CommandLine;
```

Comment:
It looks like  only whitespace/ordering changes are left in this file — can you revert them?